File: domein/src/main/java/be/genealogie/domein/dto/OudersVanKindDto.java
package be.genealogie.domein.dto;

import lombok.Data;

@Data
public class OudersVanKindDto {
    private NieuwNatuurlijkPersoonDTO vader;
    private NieuwNatuurlijkPersoonDTO moeder;
    private NatuurlijkPersoonDTO kind;
}

File: frontend/src/app/model/ouders-van-kind-dto.ts
import {NatuurlijkPersoonDto} from "./natuurlijk-persoon-dto";

export class OudersVanKindDto {
  public constructor(
    public moeder:NatuurlijkPersoonDto,
    public vader:NatuurlijkPersoonDto,
    public kind:NatuurlijkPersoonDto|undefined
  ) {
  }
}

